### ISSUE TITLE
Bludit 3.14.0 installation issues

### ISSUE THREAD

**Issue by prbt2016** [bludit/bludit#1454]
### Describe your problem
I was in the process of manual installation of Bludit 3.14.0 on Centos 7 with PHP 5.6, Apache 2.2. 

However I get the following error after entering admin username and password and clicking on install i.e : 

`Parse error: syntax error, unexpected '>' in /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/boot/rules/60.plugins.php on line 124`

i.e this code `return $a->position() <=> $b->position();`

Seems this is due to '<=>' spaceship operator which works on PHP 7.0 

https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.spaceship-op

Are the PHP requirements changed?. 

PHP 5.6 is mentioned in official site docs https://docs.bludit.com/en/getting-started/requirements.

Also when I checked install on PHP 7.0 to 7.2 on same server the admin panel throws following notice and warnings and also after entering credentials , login doesn't work, URL is redirected back to http://URL/bludit/admin/login

(For your kind info I have `display_errors = Off` in my php.ini file for all of the PHP test environments above) :

```
Notice: Array to string conversion in /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/helpers/session.class.php on line 25

Warning: session_start(): Cannot send session cookie - headers already sent by (output started at /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/helpers/session.class.php:25) in /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/helpers/session.class.php on line 36

Warning: session_start(): Cannot send session cache limiter - headers already sent (output started at /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/helpers/session.class.php:25) in /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/helpers/session.class.php on line 36

Warning: Cannot modify header information - headers already sent by (output started at /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/helpers/session.class.php:25) in /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/boot/rules/99.header.php on line 3

Warning: Cannot modify header information - headers already sent by (output started at /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/helpers/session.class.php:25) in /{{PATH}}/{{TO}}/{{BLUDIT}}/bl-kernel/boot/rules/99.header.php on line 4
```

![image](https://user-images.githubusercontent.com/17871448/188844179-d2b43bca-1210-4e3f-ad92-183c55719c6b.png)

However, Bludit works fine on PHP 7.3 without any such warnings or notices and also admin panel loads fine and after logging dashboard loads fine without any issues.

Could you please replicate the above issues at your end and fix the same?.

### Steps to reproduce the problem

- Install Bludit on PHP 5.6 syntax error is thrown.
- Install Bludit on PHP 7.0 to 7.2 admin panel is loaded with notice and warnings and login doesn't work.

### Bludit version
3.14.0

### Hosting or Webserver name
Apache/2.2.34 

### PHP version
5.6-7.2

**Comment by dignajar** [comment]
Hi, Fixed on Bludit v3.14.1.
Thank you for the report!

**Comment by prbt2016** [comment]
Hello @dignajar ,

Thanks. Shall check out 3.14.1.

Regards.
